guide/README.md

```diff
@@ -23,5 +23,9 @@ If you don't know JS but would like to learn about it, here are a few links to h
 * [Nodeschool, for both JavaScript and Node.js lessons](https://nodeschool.io/)
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
+* [Grasshopper, a JavaScript learning app by Google](https://grasshopper.app)
```

Comment:
❌ Seems to be aimed at very young audiences to the point where it's not all that useful as preparation to writing a node.js backend application (aka: discord bot)